pincer/objects/guild/guild.py

```diff
@@ -482,6 +484,222 @@ async def modify_member(self, _id: int, **kwargs) -> GuildMember:
         )
         return GuildMember.from_dict(construct_client_dict(self._client, data))
 
+      
+    @overload
+    async def create_channel(
+        self, 
+        name: str,               
+        type: Optional[int] = None,
+        topic: Optional[str] = None,
+        bitrate: Optional[int] = None,
+        user_limit: Optional[int] = None,
+        rate_limit_per_user: Optional[int] = None,
+        position: Optional[int] = None,
+        permission_overwrites: Optional[List[Overwrite]] = None,
+        parent_id: Optional[Snowflake] = None,
+        nsfw: Optional[bool] = None
+    ) -> Channel:
+        """|coro|
+        Create a new channel object for the guild.
+
+        Parameters
+        ----------
+        name : str
+            channel name (1-100 characters)
+        type : Optional[:class:int`]
+            the type of channel
+        topic : Optional[:class:str`]
+            channel topic (0-1024 characters)
+        bitrate : Optional[:class:`int`]
+            the bitrate (in bits) of the voice channel (voice only)
+        user_limit : Optional[:class:`int`]
+            the user limit of the voice channel (voice only)
+        rate_limit_per_user : Optional[:class:`int`]
+            amount of seconds a user has to wait before sending another message (0-21600)
+            bots, as well as users with the permission manage_messages or manage_channel, are unaffected
+        position : Optional[:class:`int`]
+            sorting position of the channel
+        permission_overwrites : Optional[List[:class:`~pincer.objects.guild.overwrite.Overwrite`]]
+            the channel's permission overwrites
+        parent_id : Optional[:class:`~pincer.utils.snowflake.Snowflake`]
+            id of the parent category for a channel
+        nsfw : Optional[:class:`bool`]
+            whether the channel is nsfw
+
+        """
+        ...
```

Comment:
Missing return type in the docstring, also a redundant line
```suggestion
            id of the parent category for a channel
        nsfw : Optional[:class:`bool`]
            whether the channel is nsfw
        """
        ...
```